app/javascript/chat/channelRequest.jsx

```diff
@@ -25,15 +25,15 @@ const ChannelRequest = ({ resource: data, handleJoiningRequest }) => (
       {data.channel.status !== 'joining_request' ? (
         <button
```

Comment:
refactor (blocking): @lisasy, since we're doing the refactor and this is a Preact component, you can use our `<Button />` component instead of an HTML button. This applies to anywhere in your refactor where the change was made in a Preact component.